5_data_munge/src/munge_data_files.R

```diff
@@ -40,74 +40,144 @@ munge_ecosheds <- function(in_ind, min_value, max_value, out_ind) {
            mean < max_value)
 
   message(nrow(dat) - nrow(dat_out), ' temperature observations dropped because they were outside of 0-35 deg C.')
-  
+
   dat_out2 <- dat_out %>%
     filter(!grepl('delete data|inaccurate|setup|bad value|preplacement|dewatered|out of water|out ofwater|outofwater|removal|set up|too warm', comment, ignore.case = TRUE))
-  
+
   message(nrow(dat_out) - nrow(dat_out2), ' temperature observations dropped due to comments that indicated poor data quality.')
-  
-  saveRDS(dat_out2, file = as_data_file(out_ind))  
+
+  saveRDS(dat_out2, file = as_data_file(out_ind))
+  gd_put(out_ind)
+
+}
+
+munge_norwest <- function(in_ind, min_value, max_value, out_ind) {
+  files <- yaml::read_yaml(in_ind)
+  in_ind <- as_ind_file(names(files)[1])
+  sites_ind <- as_ind_file(names(files)[2])
```

Comment:
if you make the suggested additions to 4_other_sources.yml, you shouldn't need to do this tailored partitioning in this file - you can just pass in the two indicator filenames as separate arguments.